dev/ProgressBar/TestUI/ProgressBarPage.xaml

```diff
@@ -95,7 +95,7 @@
                 <TextBlock Text="States"/>
                 <CheckBox x:Name="ShowPausedCheckBox" AutomationProperties.Name="ShowPausedCheckBox" Content="ShowPaused" />
                 <CheckBox x:Name="ShowErrorCheckBox" AutomationProperties.Name="ShowErrorCheckBox"  Content="ShowError"/>
-                <CheckBox x:Name="ShowIsDeterminateCheckBox" AutomationProperties.Name="ShowIsDeterminateCheckBox" Content="IsDeterminate"/>
+                <CheckBox x:Name="ShowIsIndeterminateCheckBox" AutomationProperties.Name="ShowIsIndeterminateCheckBox" Content="IsIndeterminate"/>
```

Comment:
You will need to update the following line too to ensure that tests won't fail since they are looking for that checkbox.

https://github.com/microsoft/microsoft-ui-xaml/blob/1ced221991d3abd51875d24d69221afa878b6c5a/dev/ProgressBar/InteractionTests/ProgressBarTests.cs#L202